Rename Firing range Cameras to 'East' and 'West'

Lorwp opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

-Fixes West Firing range camera being named 'Medical Station'

~~Changelog will come in a minute....~~

mustafakalash reviewed (changes requested):
Looks like you need to use the mapmerge tools.

Lorwp commented:
@mustafakalash doing now. Due to a semi-related issue on my first attempt to fix this, i forgot to do mapmerge when doing c0805690f6898e0e94eb32d99cbb53a243794ad9

Commit:
Rename Firing range Cameras to 'East' and 'West'

Attempt two...
-Fixes West Firing range camera being named 'Medical Station'

Signed off by lorwp, <lorwpp@gmail.com>

Commit:
Add Changelog

Lorwp commented:
Everything (should) be 👍 Now.

Lorwp commented:
If Possible, Squash and merge.

PsiOmegaDelta commented:
I prefer having the PR and repository commit history match.
